repository/BaselineOfGrease.package/BaselineOfGrease.class/instance/projectClass.st

```diff
@@ -0,0 +1,4 @@
+accessing
+projectClass
+  Smalltalk at: #'MetacelloCypressBaselineProject' ifPresent: [ :cl | ^ cl ].
```

Comment:
Okay, Google was my friend. Now I suddenly feel like I need to run over all my baselines and add it :)

Reply:
Hi @jbrichau ,
This line allows Metacello to update a metadata-less project in case the project is already loaded in the image.

Before, it used the metadata to know when to update. Since there are metadata-less formats, another way need to be used.